app/src/main/java/com/tari/android/wallet/ui/activity/qr/QRScannerActivity.kt

```diff
@@ -30,7 +30,7 @@
  * OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
```

Comment:
Not listed here though relevant, 

Looks like there is a bug here:

```
codeScanner.decodeCallback = DecodeCallback {
            val intent = Intent()
            intent.putExtra(EXTRA_QR_DATA, it.text)
            setResult(Activity.RESULT_OK, intent)
            finish()
            overridePendingTransition(0, R.anim.slide_down)
        }
```

1) Code does not appear to lead anywhere.
2) 
```
intent.putExtra(EXTRA_QR_DATA, it.text)
```

```it.text``` needs to be further processed since it may not just be emojis, i.e ```it.text.split("/").last()```

Reply:
@StriderDM You're right, thanks, the result doesn't get processed, and the deep link format is not published at OS level yet, will open an issue & be working on it.